src/main/java/com/mosaic/Weatherforecast.java

```diff
@@ -0,0 +1,40 @@
+package com.mosaic;
+
+import java.time.LocalDateTime;
+
+public class Weatherforecast {
```

Comment:
It seems you have added two of these classes.
Another one is a sub-class inside Apiresponse.

is it on purpose?